ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -721,6 +721,20 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
+- as: aws-ipi-ovn-hypershift-fips-dynamic-dns-f28
```

Comment:
suggest running it for f7 since it is a new feature in 4.19

Reply:
Or we can merge it into an existing job: 
aws-ipi-ovn-hypershift-guest-advanced-f7

Reply:
The cron job should generate by the ./ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh
i.e.,
./ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh aws-ipi-ovn-hypershift-guest-longduration-f14 openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml
cron: 13 18 4,18 * *